pyctcdecode/language_model.py

```diff
@@ -271,6 +287,84 @@ def score(
         lm_score = self.alpha * lm_score * LOG_BASE_CHANGE_FACTOR + self.beta
         return lm_score, end_state
 
+    @property
+    def serializable_attrs(self) -> Dict[str, Any]:
+        """Get a dictionary of the attributes to serialize to json."""
+        json_attrs = {}
+        for attr in LanguageModel.json_attrs:
+            val = getattr(self, attr)
+            if val is None:
+                raise ValueError(f"attribute {attr} not found. Cannot serialize")
+            json_attrs[attr] = val
+        return json_attrs
+
+    def save_to_dir(self, filepath: str) -> None:
+        """Save to a directory."""
+        json_attrs = self.serializable_attrs
+        json_attr_path = os.path.join(filepath, "attrs.json")
+        unigrams_path = os.path.join(filepath, "unigrams.json" "")
```

Comment:
extra quotes typo?
maybe more canonical to save unigrams as newline separated txt file?

Reply:
yes, typo. 
ill move to save unigrams in a txt file